docs/source/troubleshooting.md

```diff
@@ -18,7 +18,7 @@ If you have tried to start the JupyterHub proxy and it fails to start:
 recent version of [Node](https://nodejs.org). Some versions of Ubuntu/Debian come with a version
 of Node that is very old, and it is necessary to update Node.
 
-### sudospawner fails to run
+### Sudospawner fails to run
```

Comment:
`sudospawner` is actually a command, so shouldn't be Sentence-cased. You could put it in backticks for code style, though I think in headings it's also nice to not use code styling.